linux/RNGEntropy.c

```diff
@@ -0,0 +1,83 @@
+/*
+htop - RNGEntropy.c
+(C) 2022 Loic Jourdheuil
+Released under the GNU GPLv2+, see the COPYING file
+in the source distribution for its full text.
+*/
+
+#include "linux/RNGEntropy.h"
+
+#include "CRT.h"
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <unistd.h>
+#include <sys/statfs.h>
+#include <sys/statvfs.h>
+
+#include "Object.h"
+#include "XUtils.h"
+
+static void getRNGEntropy(Meter* this) {
+   FILE* file = fopen(PROCDIR "/sys/kernel/random/entropy_avail", "r");
+   if (!file)
+      return;
+   int match = fscanf(file, "%le", &this->values[0]);
```

Comment:
"%le", seriously?
I expect reading an integer and then converting to a floating point (for storing in `this->values[n]`). This would make the code more robust.
Also, don't skip the return value check, you probably need to set a placeholder value in case of read errors. Something like this:
```
if (!match)
   this->values[0] = NAN;
```